manifests/k/KiCad/KiCad/7.0.5/KiCad.KiCad.locale.en-US.yaml

```diff
@@ -19,92 +19,5 @@ Tags:
 - cad
 - eda
 - pcb

```

Comment:
```suggestion
- pcb
ReleaseNotes: |-
  Changelog
  General
  - Initialize CURL library for thread safety.
  - Fix bitmap rotation and mirroring issues.
  - Do not close dialogs on escape key when typing in a text control. \#14514
  - Do not display additional space before subscript/superscript text in  text boxes. \#14683
  - Make control-backspace clear until last whitespace in text edit controls. \#13910
  - Catch unhandled exception at startup. \#14703
  - Use correct alpha blending when plotting to PDF. \#14238
  - Fix broken bug report URL. \#14759
  Schematic Editor
  - Do not collapse tree when filtering symbol chooser dialog search results. \#13905
  - Do not ignore "Exclude from BOM" and "Exclude from board" attributes for derived symbols. \#13740
  - Correctly handle symbols with / character in the name. \#14057
  - Fix crash when pasting then pressing escape. \#14315
  - Ignore empty field name templates. \#14552
  - Ensure item is on grid if there was no motion between clicks.
  - Fix clipped text in ERC dialog. \#14569
  - Save worksheet file name in project file. \#14219
  - Allow hex entry of 0 transparency in color picker. \#14646
  - Fix datasheet path resolution when path contains double slashes. \#14564
  - Set focus to text control when editing text box properties. \#14685
  - Fix for mouse stuck in canvas issue. \#12653
  - Handle user defined attributes when importing Eagle schematics. \#13798
  - Fix broken nets using hierarchical labels. \#14657
  - Remove extra blank line in schematic file when saving with ERCmarkers. \#11405
  - Fix crash when entering subsheets with resolved variables. \#14720
  - Mirror netclass labels correctly. \#14758
  Spice Simulator
  - Do not show both sets of pins for symbols with alternate body styles (DeMorgan).\#14230
  - Do not create SPICE netlists with spaces in net names. \#14724
  Symbol Editor
  - Save library when it’s selected in library tree panel. \#14220
  - Cmd+A (Select All) selects all symbol editor components instead text in active text input box. \#14237
  - Fix crash when infobar outlives the current symbol. \#14680
  Board Editor
  - Highlight drilled hole to hole clearance of same nets in router. \#12781
  - Fix graphical glitch after DRC and subsequent undo action. \#13446
  - Do not detect DRC error when the footprint was deleted or updated with new one. \#14351
  - Allow setting board minimum widths to zero. \#13822
  - Fix crash when differential pair routing. \#14537
  - Fix invalid overbar text for escaped characters. \#14553
  - Fix overbar when using subscript characters. \#13449
  - Trim VRML export layers to board outline. \#14557
  - Handle multiple holes in footprints. \#14468
  - Do not treat via placement as DRC violation when routing. \#14603
  - Record parent rule for DRC assertions.
    See forum discussion
  - Fix incorrect mounting hole size on DSN export. \#14656
  - Allow double-clicking item to open properties dialog in search pane. \#12475
  - Fix crash when loading a file having vias on connected layers only. \#14668
  - Add missing property to footprint texts table. \#14357
  - Fix copper sliver issues. \#14549
  - Fix issue routing differential pairs to footprint pads. \#14324
  - Expand solder mask for NPTH pads. \#14693
  - Fix field automatic placement issue. \#14127
  - Don’t allow thickness of 0 in STEP export in case of an empty stackup. \#10790
  - Make footprint drag include connected traces.\#14515
  - Add missing property to footprint texts table. \#14357
  - Use constraints to determine netclass width. \#14190
  - Fix rotating bitmaps.\#14197
  - Fix shove router clearance DRC violations. \#14707
  - Do not route trans with clearance errors. \#14659
  - Fill shapes with open path when importing SVG. \#14518
  - Ensure track posture switches on first attempt. \#12369
  - Prevent unwanted minus sign in properties panel. \#14168
  - Adjust stroke font output to match version 6 output. \$14609
  - Fix crash on start of routing. \#14733
  - Fix crash when shoving via or trace. \#14741
  Footprint Editor
  - Crash when importing version 5.1 settings. \#14691
  Gerber Viewer
  - Fix rounded corners when exporting to KiCad board file. \#14136
  3D Viewer
  - Use correct board stack up layer color settings. \#14616
  - Show footprint board regardless of project. \$14246
  PCB Calculator
  - Save regulator calculator settings.
  Drawing Sheet Editor
  - Add missing grid properties dialog. \#14612
  Command Line Interface
  - Load custom drawing sheets for board. \#14171
  - Do not fail silently if output directory does not exist. \#14437
  Windows
  - Fix broken text selection for some UI elements. \#14692
ReleaseNotesUrl: https://github.com/KiCad/kicad-source-mirror/releases/tag/7.0.5
```